queue_services/entity-filer/tests/unit/test_worker/test_correction_bcia.py

```diff
@@ -407,49 +419,49 @@ async def test_worker_correction_court_order(app, session, mocker, test_name, le
 @pytest.mark.parametrize(
     'test_name',
     [
-        'ben_add_director',
-        'ben_edit_director_name_and_address',
-        'ben_delete_director',
+        'bc_add_director',
+        'bc_edit_director_name_and_address',
+        'bc_delete_director',
```

Comment:
can we update this test to test all the relevant entity types?  i.e. BEN, BC, ULC, CC